DollyCamPlugin2/dollycam.cpp

```diff
@@ -146,29 +146,29 @@ void DollyCam::Apply()
 	{
 		return;
 	}
-	if (currentFrame < currentPath->begin()->first || currentFrame > (--currentPath->end())->first)
+	if (currentFrame < currentPath->begin()->first || currentFrame >(--currentPath->end())->first)
 		return;
 	//cvarManager->log("Frame: " + to_string(currentFrame) + ". Replay time: " + to_string(sw.GetReplayTimeElapsed()));
 	if (currentFrame == currentPath->begin()->first)
 	{
 		if (isFirst) {
 			diff = sw.GetSecondsElapsed();
-			
+
 			isFirst = false;
 		}
 	}
 	else {
 		isFirst = true;
 	}
-/*	auto replayServerWrapper = gameWrapper->GetGameEventAsReplay();
-	auto replay = gameWrapper->GetGameEventAsReplay().GetReplay();
-	float t = sw.GetSecondsElapsed() - diff + currentPath->begin()->second.timeStamp;
-	float timeFromFrames = replayServerWrapper.GetCurrentReplayFrame() / replay.GetRecordFPS();
-	float t2 = replayServerWrapper.GetReplayTimeElapsed();
-	float t3 = sw.GetReplayDirector().GetReplayTimeSeconds();
-	cvarManager->log("gameTime:" + to_string(t) + ", frameTime: " + to_string(timeFromFrames) + ", GetReplayTimeSeconds:" + to_string(t3) + ", replayServerTime:" + to_string(t2));
-	*/
-	//cvarManager->log("gameTime:" + to_string(t) + ", diff: " + to_string(diff) + ", sw:" + to_string(sw.GetSecondsElapsed()) + ", begin:" + to_string(currentPath->begin()->second.timeStamp) + ", frameTime: " + to_string(timeFromFrames));
+	/*	auto replayServerWrapper = gameWrapper->GetGameEventAsReplay();
```

Comment:
could you remove this?

Reply:
Sure, will remove on next PR that creates files inside a dollycam directory inside bakeksmod folder instead of bloating Win32